appholder/src/main/java/com/android/identity/wallet/util/PreferencesHelper.kt

```diff
@@ -87,15 +87,15 @@ object PreferencesHelper {
     }
 
     fun isConnectionAutoCloseEnabled(): Boolean {
```

Comment:
nit: single line functions where the return type is clear in kotlin are often better written as: 

fun isConnectionAutoCloseEnabled() = sharedPreferences.getBoolean(CONNECTION_AUTO_CLOSE, true)

if you wanted to be fancy you could also do this as a var with overridden getter/setters:

var connectionAutoCloseEnabled: Boolean
  get() = sharedPreferences.getBoolean(CONNECTION_AUTO_CLOSE, true)
  set(enabled) { sharedPreferences.edit { putBoolean(CONNECTION_AUTO_CLOSE, enabled) } }

Reply:
Cool. I didn't write that code but I'll keep that in mind (I'm still a Kotlin novice so this is helpful!)